examples/units_and_coordinates/PSP_trajectory_sunpy.py

```diff
@@ -0,0 +1,45 @@
+"""
+============================================
+Getting the trajectory of Parker Solar Probe
+============================================
+
+This example shows how to plot the trajectory of Parker Solar Probe (PSP) within the solar system.
+"""
+
+import matplotlib.pyplot as plt
+
+from sunpy.time import parse_time
+
+from sunpy.coordinates import get_body_heliographic_stonyhurst
+from sunpy.coordinates.ephemeris import get_horizons_coord
+
+##############################################################################
+# By using `~sunpy.coordinates.ephemeris.get_horizons_coord`, we can query the 
+# Jet Propulsion Laboratory `Horizons System <https://ssd.jpl.nasa.gov/horizons/>`__ 
+# to get the location of any body they support, but here we will use it to get
+# the coordinates of Parker Solar Probe.
+
+now = parse_time("now")
+trajectory_coords = get_horizons_coord('Parker Solar Probe', {'start': '2021-10-11', 'stop': '2022-01-12', 'step': '180m'})
```

Comment:
I think we should be using the date for now - 3 months and by now + 3 months

Otherwise we should fix the date instead of using now.